functions/tests/test_stripe_create_account.py

```
@@ -0,0 +1,451 @@
+        # Mock transaction and organiser reference
+        mock_transaction = Mock()
+        mock_db.transaction.return_value = mock_transaction
+        mock_organiser_ref = Mock()
+        mock_db.collection.return_value.document.return_value = mock_organiser_ref
+
+        mock_check_update.return_value = json.dumps(
+            {"url": "https://connect.stripe.com/test"}
+        )
+
+        with patch("lib.stripe.create_account.uuid.uuid4", return_value="test-uuid"):
+            result = create_stripe_standard_account(mock_request)
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)